orbyter-ml-dev/Dockerfile

```diff
@@ -1,30 +1,13 @@
-# orbyter-ml-dev:3.5
-FROM manifoldai/orbyter-base-sys:3.2
+# orbyter-ml-dev:4.0
+FROM manifoldai/orbyter-base-sys:4.0
 COPY . /build
 WORKDIR /build
-RUN apt-get update && \
-    apt-get install -y libpq-dev && \
-    apt-get install -y graphviz && \
+RUN apt-get update -qq && \
+    apt-get install -qqy --no-install-recommends \
+        libpq-dev \
+        graphviz && \
     pip install -r requirements.txt && \
-    jupyter contrib nbextension install --user && \
-    jupyter nbextension enable --py widgetsnbextension && \
```

Comment:
What's the reasoning behind removing the notebook widget stuff?

Reply:
The reason for removing the `jupyter contrib nbextension install` command is because according to the docs [here](https://github.com/ipython-contrib/jupyter_contrib_nbextensions), the extensions are meant for Jupyter Notebook, not JupyerLab. The reason for removing the widget stuff is due to the text in the docs [here](https://ipywidgets.readthedocs.io/en/latest/user_install.html) that state this command is only needed for older versions of Jupyter Notebook.

Though if this _does_ turn out to actually break something for anyone, please let me know.